src/main/java/lotto/model/LottoTickets.java

```diff
@@ -27,4 +28,8 @@ public int getSize() {
     public List<LottoTicket> getLottoTickets() {
         return Collections.unmodifiableList(lottoTickets);
     }
+
+    public void add(List<LottoTicket> additionalLottoTickets) {
+        lottoTickets.addAll(additionalLottoTickets);
+    }
```

Comment:
동일한 `LottoTickets` 타입에 대해서 add 메서드를 제공해보면 어떨까요? 🙂
```suggestion
    public void add(Lottos lottoTickets) {
        ....
    }
```

Reply:
말씀주신 add 메서드 추가하고 관련 테스트도 추가하였습니다.